packages/snap-api/src/api/benchmark-zkp-gen.ts

```diff
@@ -8,7 +15,10 @@ import { invokeSnap } from '../utils/invoke-snap';
  */
 export type BenchmarkZKPGenParams = {
   // Input for the ZKP generation.
-  input: Record<string, unknown>;
+  input: PreparedZkCertProofInputs<
```

Comment:
For the benchmark the input is not required to have all the certificate fields. Setting it to just `Record<string, FieldElement | FieldElement[]>` also removes the need for `as any` in the unittest.

Reply:
Got it, thanks. But it is a concerning idea to have a benchmark as part of user-facing API. Can we avoid it? It can be dangerous, because it increases API surface, which can be viewed as increased attack surface.